src/test/java/org/tugraz/sysds/test/functions/builtin/BuiltinSTEPLmTest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2018 Graz University of Technology
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tugraz.sysds.test.functions.builtin;
+
+import org.junit.Test;
+import org.tugraz.sysds.common.Types.ExecMode;
+import org.tugraz.sysds.lops.LopProperties.ExecType;
+import org.tugraz.sysds.runtime.matrix.data.MatrixValue.CellIndex;
+import org.tugraz.sysds.test.AutomatedTestBase;
+import org.tugraz.sysds.test.TestConfiguration;
+import org.tugraz.sysds.test.TestUtils;
+
+import java.util.HashMap;
+
+
+public class BuiltinSTEPLmTest extends AutomatedTestBase{
```

Comment:
Please change the indentation from spaces to tabs (we use for historic reasons tabs for code, and spaces for dml scripts).